main.js

```diff
@@ -1,10 +1,10 @@
 
-import { main_BANG_ } from "./js-out/quamolit.main.js"
+import { main_$x_ } from "./js-out/quamolit.app.main.js"
```

Comment:
main_$x_ ?

Reply:
keep in mind that calcit-js is a Lisp dialect and in Lisp, it's possible to have `main!` as a function name, according to change in https://github.com/calcit-lang/calcit_runner.rs/pull/56 we have `!` compiled to `_$x_` and now we got this.